Twitter/Controllers/V1/PostController.cs

```diff
@@ -40,12 +42,37 @@ public async Task<ActionResult<List<Post>>> Get()
 			return Ok(await _repository.Post.FindAllPostsAsync());
 		}
 
+		/// <summary>
+		/// Returns posts with pagination.
+		/// </summary>
+		/// <returns>Returns posts with pagination.</returns>
+		/// <response code = "200">Returns a list of posts with pagination.</response>
+		/// <response code = "404">Throws an exception if posts don't exist.</response>
```

Comment:
If we are reading posts, there can be only 0 posts, not NotFound.

Reply:
I've fixed it